ModelBundle/Repository/NodeRepository.php

```diff
@@ -189,6 +190,22 @@ public function findByParentIdAndSiteId($parentId, $siteId)
     }
 
     /**
+     * @param string $path
+     * @param string $siteId
+     *
+     * @throws \Doctrine\ODM\MongoDB\MongoDBException
+     *
+     * @return mixed
+     */
+    public function findByPathAndSiteId($path, $siteId)
```

Comment:
The name findByPathAndSiteId suggests it will return a single node, the one matching $path. Maybe it could be renamed into findSubreeByPathAnsSiteId or something like this